Cargo.toml

```diff
@@ -7,16 +7,15 @@ build = "build.rs"
 description = "A simple GUI for Neovim."
 
 [features]
-default = ["sdl2-static-link", "embed-fonts"]
-sdl2-static-link = ["skulpin/sdl2-static-link"]
+default = ["embed-fonts"]
 embed-fonts = []
 
 [dependencies]
 euclid = "0.20.7"
 font-kit = "0.5.0"
 skribo = { git = "https://github.com/linebender/skribo" }
 lru = "0.4.3"
-skulpin = { git = "https://github.com/Kethku/skulpin", branch = "sdl2" }
+skulpin = {version = "0.9.0", features=["skulpin_sdl2"]}
```

Comment:
rumor has it we can add back the sdl2-static-link by overriding the sdl2 dependency. We should probably figure out how that works before merging this

Reply:
I usually add the dependency to [patch.crates-io]. That's how I fixed my build errors in my game project when integrating neovide.

Reply:
That makes sense. Can you specify features that way?

Reply:
Yep, it's pretty much exactly the same as specifying any other dependency to my knowledge.

Reply:
e.g.
```toml
[patch.crates-io]
# [patch.crates-io.sdl2-sys]
sdl2 = { git = "https://github.com/Rust-SDL2/rust-sdl2.git" }
# # path = "../rust-sdl2/sdl2-sys"
sdl2-sys = { git = "https://github.com/Rust-SDL2/rust-sdl2.git" } 
```

Reply:
very cool